src/common/components/ui/OpenShiftVersionDropown.tsx

```diff
@@ -82,17 +86,26 @@ export const OpenShiftVersionDropdown = ({
 
   return (
     <FormGroup fieldId={fieldId} label={t('ai:OpenShift version')} isRequired>
-      <Dropdown
-        {...field}
-        name={name}
-        id={fieldId}
-        onSelect={onSelect}
-        dropdownItems={dropdownItems}
-        toggle={toggle}
-        isOpen={isOpen}
-        className="pf-u-w-100"
-      />
-      <HelperText style={{ display: 'inherit' }}>{helperText}</HelperText>
+      <Flex direction={{ default: 'column', lg: 'row' }}>
+        <FlexItem flex={{ default: 'flex_1' }}>
+          <Dropdown
+            {...field}
+            name={name}
+            id={fieldId}
+            onSelect={onSelect}
+            dropdownItems={dropdownItems}
+            toggle={toggle}
+            isOpen={isOpen}
+            className="pf-u-w-100"
+          />
+          <HelperText style={{ display: 'inherit' }}>{helperText}</HelperText>
+        </FlexItem>
+        <FlexItem align={{ default: 'alignRight' }}>
+          <ExternalLink href={`${window.location.origin}/${OCP_RELEASES_PAGE}`}>
```

Comment:
About the position of the link, is not possible to put outside of the form.
Maybe we can put under the field. @nirfarkas WDYT?

Reply:
I think @jgyselov is right.

Since the component is in `common`, it should only display the link when passed a prop. 
Add a `addReleaseLink` prop (based on `ocmClient` as suggested), and on OCM we'll set it to true, then if ACM want to use it, they can pass the prop as needed.

Reply:
Ok, I add the condition ocmClient to show the link.
I can try to change the position of the link, If not I put below the field.

Reply:
Finally I put the link below the field.